src/components/panel/panel.js

```diff
@@ -1723,21 +1851,27 @@ MdPanelPosition.prototype.absolute = function() {
 };
 
 /**
- * Sets the value of a position for the panel. Clears any previously set position.
+ * @description
+ * Sets the value of a position for the panel. Clears any previously set
+ * position.
+ *
  * @param {string} position Position to set
  * @param {string=} value Value of the position. Defaults to '0'.
- * @returns {MdPanelPosition}
+ * @returns {!MdPanelPosition}
  * @private
  */
 MdPanelPosition.prototype._setPosition = function(position, value) {
-  if (position === MdPanelPosition.absPosition.RIGHT || position === MdPanelPosition.absPosition.LEFT) {
-    this._left = this._right = '';
-  }
-  else if (position === MdPanelPosition.absPosition.BOTTOM || position === MdPanelPosition.absPosition.TOP) {
-    this._top = this._bottom = '';
-  }
+  if (position === MdPanelPosition.absPosition.RIGHT ||
+      position === MdPanelPosition.absPosition.LEFT) {
+        this._left = this._right = '';
+      }
+  else if (position === MdPanelPosition.absPosition.BOTTOM ||
+      position === MdPanelPosition.absPosition.TOP) {
```

Comment:
To end all discussion with this method, here's what the formatter does and how we would write it. Note the indentation for the if statements.

If we want, I can run the formatter across the entire file on it's own PR and fix all the spacing issues.

```
MdPanelPosition.prototype._setPosition = function(position, value) {
  if (position === MdPanelPosition.absPosition.RIGHT ||
      position === MdPanelPosition.absPosition.LEFT) {
    this._left = this._right = '';
  } else if (
      position === MdPanelPosition.absPosition.BOTTOM ||
      position === MdPanelPosition.absPosition.TOP) {
    this._top = this._bottom = '';
  } else {
    var positions =
        Object.keys(MdPanelPosition.absPosition).join().toLowerCase();
    throw new Error('Position must be one of ' + positions + '.');
  }

  this['_' + position] = angular.isString(value) ? value : '0';
  return this;
};
```